decidim-admin/app/packs/stylesheets/decidim/admin/_table-list.scss

```diff
@@ -108,6 +108,16 @@
   .icon-active {
     @apply bg-[#5dc560] inline-block w-3 h-3 align-middle mr-2 rounded-[50%];
   }
+
+  .draggable-table {
+    &.dragging {
+      @apply border-dashed border-y-2 border-gray;
+    }
+
+    .dragging-handle {
+      @apply cursor-move;
+    }
+  }
```

Comment:
Using the API from https://github.com/decidim/decidim/pull/13112, this change would not be needed anymore.

Reply:
Following conversation in [this comment](https://github.com/decidim/decidim/pull/13074#discussion_r1740626447)